backend/src/v5/models/tickets.js

```diff
@@ -34,77 +32,102 @@ const determineTicketNumber = async (teamspace, project, model, type) => {
 	return (lastTicket?.number ?? 0) + 1;
 };
 
-Tickets.addTicket = async (teamspace, project, model, ticketData) => {
-	const _id = generateUUID();
-	const number = await determineTicketNumber(teamspace, project, model, ticketData.type);
-	const ticket = { ...ticketData, teamspace, project, model, _id, number };
-	await DbHandler.insertOne(teamspace, TICKETS_COL, ticket);
-	publish(events.NEW_TICKET,
-		{ teamspace,
-			project,
-			model,
-			ticket: { ...ticketData, _id, number } });
-	return _id;
+// We expect all tickets to have the same template (i.e. type field should be the same in all tickets provided)
+Tickets.addTicketsWithTemplate = async (teamspace, project, model, templateId, tickets) => {
```

Comment:
Initially I called it something like that, but I added WithTemplate in the end to ensure that the user is not trying to do insert of different ticket types. 

So not sure if I agree, could call it `addTicketsWithSameTemplate` if you want but I think the meaning and the place in code (if we're adding both this would be a processor function as templates would be in a different db model) is pretty clear.